azure-pipelines.windows.yml

```diff
@@ -0,0 +1,147 @@
+name: Roaster Windows Build
+
+trigger:
+- master
+
+pool:
+  vmImage: 'windows-latest'
```

Comment:
vmImage doesn't work for current selfhost windows agent pool.

should directly use the pool name here.